### PR TITLE
2024/XenSummit/cfp: Add CFP for presentation

### PR DESCRIPTION
**Pull request by miczyg1**

### CODE REVIEW

File: 2024/XenSummit/cfp/trenchboot.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+Challenges and status of enabling TrenchBoot in Xen hypervisor
+
+## Description
+
+The presentation will briefly describe the recent development and progress of
+enabling TrenchBoot in Xen hypervisor, the challenges we had to overcome and
+resulting changes to the Xen hypervisor project. The security properties of a
+system running Xen with TrenchBoot support will be explained. A short demo of
+running DRTM with Xen hypervisor will be shown. Also, the incoming UEFI support
+and the impact on Xen UEFI boot process shall be discussed.
```
Comment (pietrushnic) [inline 2024/XenSummit/cfp/trenchboot.md:14]: @ miczyg1, please explain the stack we plan to test. I know you want to keep commitment generic, but if we will fail in given scenario we can say that and explain details during presentation. When describing stack we should mention Dasharo for those platform on which we plan to have it.

File: 2024/XenSummit/cfp/trenchboot.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+## Are you both the submitter and speaker?
+
+**Yes**/ No
+
+## Do you require travel funding?
+
+(Please note travel funding requests are not guaranteed. The Xen project will
+review all travel requests.)
+
+Yes???
```
Comment (pietrushnic) [inline 2024/XenSummit/cfp/trenchboot.md:66]: I doubt we qualify.

File: 2024/XenSummit/cfp/trenchboot.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+## Additional notes
+
+(Visible to organizers and evaluators only.)
+
+## Speaker biography
+
+Michał Żygowski is a versatile engineer with a strong focus on system
+firmware. Works as a firmware engineer at 3mdeb. Active contributor of
```
Comment (pietrushnic) [inline 2024/XenSummit/cfp/trenchboot.md:75]:
```suggestion
firmware. Works as a Firmware Engineer at 3mdeb. Active contributor of
```

**Review by pietrushnic — CHANGES_REQUESTED** [review]

File: 2024/XenSummit/cfp/trenchboot.md (hunk 1 above)
Comment (miczyg1) [reply 2024/XenSummit/cfp/trenchboot.md]: c6d89a66c724e88f4d07ba89dbc48f4a2dc0933b

Comment (pietrushnic) [reply 2024/XenSummit/cfp/trenchboot.md]: @miczyg1 much better.

**Review by pietrushnic — APPROVED** [review]

**Pull request merged**